packages/web-frontend/package.json

```diff
@@ -12,9 +12,9 @@
   "main": "dist/index.js",
   "scripts": {
     "analyze": "source-map-explorer build/static/js/main.*",
-    "build": "mkdir -p builds && npm run build-desktop && npm run build-mobile",
-    "build-desktop": "GENERATE_SOURCEMAP=false react-scripts build && rm -rf builds/desktop && mv build builds/desktop",
-    "build-mobile": "GENERATE_SOURCEMAP=false REACT_APP_APP_TYPE=mobile react-scripts build && rm -rf builds/mobile && mv build builds/mobile",
+    "build": "mkdir -p builds && npm run build-desktop && mv build builds/desktop && npm run build-mobile && mv build builds/mobile && mv builds build",
```

Comment:
I think I get the idea behind this change. But I have a feeling this might cause issues if someone goes to rebuild and runs `yarn build-desktop && yarn build-mobile` instead of `build`, it would not actually work like they expect it to.

Reply:
Yeah that's true that that case wouldn't work, but I guess I'm trying to make _most_ cases more expected (at the sacrifice of this one). So I can imagine that locally, people would generally build one or the other depending on what they were working on, and the result would be in the conventional `build` folder now. Gonna leave this one as is!